components/director/internal/gqlschema/schema.graphql

```diff
@@ -12,6 +12,8 @@ scalar HttpHeaders # -> map[string][]string
 
 scalar QueryParams # -> map[string][]string
 
+scalar EventSubscriptionAttributes # -> map[string]string
```

Comment:
Right now do not update schema itself. Diagrams are more DRAFT/Proposal for further discussions,